core/src/test/java/org/frankframework/pipes/LogContextPipeTest.java

```diff
@@ -24,6 +26,18 @@
 
 public class LogContextPipeTest extends PipeTestBase<LogContextPipe>{
 
+	@BeforeEach
+	public void setUp() throws Exception {
+		super.setUp();
+		ThreadContext.clearMap();
```

Comment:
Zou alleen deze niet voldoende moeten zijn (dus zonder tearDown)?

Reply:
Het ook in de Teardown doen beschermt de volgende test, ook als deze in een andere test-class zit. Altijd netjes om op te schonen als de test zelf niet alles goed opgeschoont heeft.

Eigenlijk zou het in de Setup niet nodig moeten zijn, maar daar doe ik het als bescherming tegen een andere test die hem misschien niet goed opgeschoond heeft.

Dus ik doe het liever 2x.

Reply:
Prima!